buildscripts/incremental/build.cmd

```diff
@@ -0,0 +1,5 @@
+
+call activate %CONDA_ENV%
+
+@rem Build numba extensions without silencing compile errors
```

Comment:
It doesn't silence errors, only informational messages.
The comment is a bit obsolete. In previous scripts we were also calling `python setup.py install > /dev/null`, which *did* silence compile errors.